website/api/helpers/get-extended-osquery-schema.js

```diff
@@ -73,6 +73,13 @@ module.exports = {
 
         expandedTableToPush.fleetRepoUrl = 'https://github.com/fleetdm/fleet/new/main/schema/tables/?filename='+encodeURIComponent('/tables/'+expandedTableToPush.name)+'.yml&value='+encodeURIComponent(sampleYamlSchemaForThisTable);
 
+        // Grab the last example in the array as is usually the most rich example.
+        // TODO: adjust the UI to show all examples.
+        let examples = expandedTableToPush.examples;
```

Comment:
Can you change the name of this variable to be clear about what it is? e.g., `examplesFromOsquerySchema`

Reply:
done!